milestone3/frontend/src/RecipePage/RecipePage.tsx

```diff
@@ -0,0 +1,54 @@
+import { useParams } from "react-router-dom"
+import { useState, useEffect } from "react";
+import { Recipe } from "../types";
+import './RecipePage.css';
+export function RecipePage() {
+  const { recipeName } = useParams();
+  const [recipe, setRecipe] = useState<Recipe | null>(null);
+  useEffect(() => {
+    async function fetchRecipe() {
+      const response = await fetch(`https://bootcamp-milestone-4.onrender.com/recipe/${recipeName}`);
+      const recipe = (await response.json() as Recipe[])[0];
+      setRecipe(recipe);
+    }
+    fetchRecipe();
+  }, [recipeName]);
```

Comment:
This is great, you're doing a good job getting the recipe from the api, however for milestone 3.5 we also want to display all of your old recipes from your milestone 3 with the recipeData.ts file. I would take another look at the [Milestone 3.5](https://h4i.notion.site/Milestone-3-5-50c906e855fe42db981e28e3dfca593f) doc and make sure you're meeting all the requirements

Reply:
I added this for 3.5, but looks like for 4 and 5 we switch back to just displaying our recipes, so the current final PR doesn't do this